Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -526,7 +526,11 @@ private final class TokenStreamCreator: SyntaxVisitor {
 
   override func visit(_ node: AsExprSyntax) {
     before(node.asTok, tokens: .space)
-    after(node.asTok, tokens: .break)
+    if let punctuationToken = node.questionOrExclamationMark {
```

Comment:
This could probably also be accomplished with `before(node.typeName, tokens: .break)`

Reply:
Good point, that's cleaner. Done.